### ISSUE TITLE
REOPENED - No 64 bit version of libandroid_platform.so native library

### ISSUE THREAD

**Issue by BeanStalka** [Azure-Samples/Cognitive-Speech-STT-Android#46]
REOPENED FROM https://github.com/Azure-Samples/Cognitive-Speech-STT-Android/issues/34 as my question was not answered.

ORIGINAL ISSUE:

_I'm using Xamarin and using a bound library to leverage the Bing Speech Recognition API
Since I have other NuGet packages that use 64 bit versions, I also need a 64 bit version of the libandroid_platform.so_.

_This is a blocker and if I cannot find away around this soon, we will have to look at other technologies for our STT needs_.
(I have a 32 Bit Library for android need a 64 bit version of the same library)

ORIGINAL RESPONSE:

_Hi - We recently released the protocol documentation so developers can write their platform/language dependent implementation. More info here : https://docs.microsoft.com/en-us/azure/cognitive-services/speech/api-reference-rest/websocketprotocol.
Hope that helps._
(Here is how you use websockets to call BING)

PROBLEM WITH RESPONSE:

I and anyone else that is using the BING Android SDK in a 64 bit environment needs to have the libandroid_platform.so library compiled for 64 bit.

I should not have to refactor my Bing SDK implementation to  manually make web socket requests in order to achieve this.  That's the whole point of a library, you save me the trouble.

Its bad enough you do not have a NuGet Package for Xamarin, but now it seems you do not have a 64 bit version of microsoft's premier Speech Recognition service?

If you do not have one, and no plans to make one please let me know. 

If there are plans to get one out there let me know.  

But please don't sell me a work around as if its a solution.

**Comment by priyaravi20** [comment]
Thanks for the feedback. The SDKs currently available are the only ones we plan to support. As I pointed out our plan is to opensource the next set of SDKs and the first in the series is what we have exposed in form of protocol and JavaScript SDK which was released in May. We do plan to add to this in multiple platforms but I do not have dates to share at this moment. So to answer your question we do not plan to release a 64 bit version but do plan to opensource SDKs for better developer support.
